src/VisualStudio/Core/Test/Progression/OverridesGraphQueryTests.vb

```diff
@@ -50,7 +50,7 @@ class Goo : Base, IComparable
                             <Link Source="(@1 Type=Goo Member=(Name=CompareTo OverloadingParameters=[(@2 Namespace=System Type=Object)]))" Target="(@1 Type=Base Member=(Name=CompareTo OverloadingParameters=[(@2 Namespace=System Type=Object)]))"/>
                         </Links>
                         <IdentifierAliases>
-                            <Alias n="1" Uri="Assembly=file:///Z:/CSharpAssembly1.dll"/>
+                            <Alias n="1" Uri="Assembly=file:///Z:/bin/CSharpAssembly1.dll"/>
```

Comment:
why were tehse changed?

Reply:
I changed the test helpers to use paths that differentiate between `obj` and `bin` dirs, so that it's clear which one is actually being expected. The IDE tracks both paths on project.